ocp4/product.yml

```diff
@@ -0,0 +1,15 @@
+product: ocp4
+full_name: Red Hat OpenShift Container Platform 4
+type: platform
+
+benchmark_root: "../applications"
+
+extra_rule_group_roots:
```

Comment:
Can this be `additional_directories`?

Reply:
Well it of course can, but I'd rather prefer it won't. 

The name is sort of a message. First of all `roots` means that it behaves like `benchmark_root` key (diving into subdirs while collecting rules etc) while being a list of these roots (hence the plural form). Next, this option is working in such a way that it will bring into base benchmark only rules, and nothing else from other benchmarks will be inherited; you can have there any directory with a group.yml file and a subdir of rules (so there is `rule_group`). And, finally, these rules will be `extra` rules, nothing would be shadowed, overridden etc, etc.

Reply:
BTW, do you want to change the behaviour along with the renaming?

Reply:
@evgenyz++, I think that @redhatrises's proposal `additional_directories` doesn't directly explain the content of the directories.

Reply:
`extra_rule_group_roots` is equally not accurate. The rules are `additional` rules not `extra` rules. There is more that just rules and groups so `rule_group` is not accurate either so it should be `content`. Finally, we are adding addition group and rule directories, and not `roots` of a benchmark which is also incorrect. `additional_content_directories` is a wayyyyyyy better key name since we are adding additional content to an existing benchmar, and documentation should explain directly in more detail what `additional_content_directories` is and does.

Reply:
As far as I know and perceive – extra == additional. And extra is a bit shorter. I don't really care, I could change it.

Then, it the middle, we have 'content', 'group' and 'rule' options. I'm non-native speaker and have not much of experience in the project, but for me 'content' word does not transmit the idea of collecting only rules from the directories in question. Can we say that rules == content? If everybody else is comfortable with it I'll change.

And finally 'root' vs 'directories'. Directories better describes what is expected here. Root links behaviour to the benchmark option (recursive scan) and also ties them together to some degree.

Again, I don't *really* care how it would be named, just trying to make most of the sense.

Reply:
Okay, `additional_content_directories` it is.